### PR TITLE
Sdk 3284

### PR DESCRIPTION
**Pull request by wiperz1789**
Has lot of commits due to rebasing and merge. Will squash everything before merging

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,5 @@ services:
       - "DISPLAY=${DISPLAY}"
 
     
-    command: "/opt/testapp-artifact/build/vision/bin/frame-detector-webcam-demo -d /opt/testapp-artifact/affectiva-ics-sdk-2.0.0/data" # you can comment this out to get a bash prompt
-    #command: '/opt/testapp-artifact/build/vision/bin/frame-detector-video-demo -d /opt/testapp-artifact/affectiva-ics-sdk-2.0.0/data -i "sample.mp4" --numFaces 50'
+   #command: '/opt/testapp-artifact/build/vision/bin/frame-detector-video-demo -d /opt/testapp-artifact/affectiva-ics-sdk-2.0.0/data -i "sample.mp4" --numFaces 50'
```
Comment (adennie) [inline docker-compose.yml:33]: seems like we don't need this file in the ics-2.2-qc branch.  Looks like it it sets up access to an x11 server for UI and a microphone for sound, but we're not doing either of those things in 2.2 QC

Comment (wiperz1789) [reply docker-compose.yml:33]: Fixed it!

Comment (adennie) [reply docker-compose.yml:33]: Oh, I meant that I don't think we need this whole docker-compose.yml file.

Comment (wiperz1789) [reply docker-compose.yml:33]: Somehow in squashing and merging this file got missed. But in new PR I removed it

**Pull request merged**